src/doc/rgw/user-accounts.md

```diff
@@ -0,0 +1,73 @@
+# iam accounts
+
+add a user account primitive for use with IAM REST APIs. this enables end-users to use tools like `aws iam create-user` for self-service management of the users, roles, policies, and access keys in their account without requiring global admin privileges for radosgw-admin commands or radosgw /admin REST APIs
+
+add new IAM APIs for User and AccessKey operations. some Role/RolePolicy/UserPolicy operations are already supported, but require special admin capabilities. once a User or Role are added to an account, that account's root user will be granted full access to them
+
+existing users and roles must be added to an account manually, either by a radosgw-admin command or /admin API, before they can be managed by an account root user
```

Comment:
@cbodley : existing rgw users have certain permissions by default like `CreateBucket` etc. What happens to those permissions when they are added as an iam user to an account?

Reply:
> @cbodley : existing rgw users have certain permissions by default like `CreateBucket` etc. What happens to those permissions when they are added as an iam user to an account?

they're lost. when you adopt users into an account, you also need to add some identity policy

Reply:
> existing resource policy (bucket policy) has ::user/* set as Principal

you can find the updated `LocalApplier::is_identity()` in https://github.com/ceph/ceph/pull/54333

currently, account users will match the following principals:
* wildcard principal `*`
* user principals `arn:aws:iam::{account_id}:user/{path}{display_name}` or `arn:aws:iam::{account_id}:user/*`
* account principals `arn:aws:iam::{account_id}:root` or `{account_id}`
* account principals `arn:aws:iam::{tenant}:root` or `{tenant}`

it probably makes sense to add matching for user principals like `arn:aws:iam::{tenant}:user/{user_id}` and `arn:aws:iam::{tenant}:user/*` for backward compatibility with existing resource policy

Reply:
the question was specifically for tenant-id/user/*, tenant was in angle brackets and didn't show up in the comment.